doc/mgr/dashboard.rst

```diff
@@ -287,7 +287,15 @@ More details can be found on the `documentation <http://docs.ceph.com/docs/maste
         grafana-cli plugins install vonage-status-panel
         grafana-cli plugins install grafana-piechart-panel
 
-#. Add the Dashboards to Grafana by importing them
+#. Add the Dashboards to Grafana by importing them::
```

Comment:
Since the following instructions will not import the grafana dashboard, I think we should either:

- Change the section title and just mention where and how we can download the dashboards.
- Add more info in this section telling how to import them.

Reply:
Ack, will add more details for importing